app/web/next-i18next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  i18n: {
+    defaultNS: "global",
+    defaultLocale: "en",
+    locales: ["en", "fr"],
```

Comment:
Not sure if we should have "fr" in here until we have a decent amount of the app translated

Reply:
That was mostly for me to test :P